VD_Transmit/VD_Transmit.ino

```diff
@@ -0,0 +1,177 @@
+/* CAN-Bus transmit vehicle dynamics node*/
+
+#include <CAN.h>
+#include <millisDelay.h>
+#include "src/can_conv/sbm_network_definition.h"
+
+#define SERIAL_DEBUG_PRINTS_ON 1
+
+/*******************************************************************************/
+
+#include <Adafruit_LSM6DSOX.h>
+Adafruit_LSM6DSOX lsm6ds;
+
+#include <Adafruit_LIS3MDL.h>
+Adafruit_LIS3MDL lis3mdl;
+
+/*******************************************************************************/
+
+/* This defines the data storage object that the generated code us//s */
+static can_obj_sbm_network_definition_h_t can_obj;
+
+// Unions are special datatypes that provide two views of the same memmory location.
+union MsgData
+{
+  uint64_t a;   // dbcc output uses uint64_t
+  uint8_t b[8]; // Adafruit CAN library wants uint8_t[8]
+};
+
+millisDelay CAN_Send_Delay; 
+
+
+void setup()
+{
+  /*******************************************************************************/
+  pinMode(PIN_CAN_STANDBY, OUTPUT);
+  digitalWrite(PIN_CAN_STANDBY, false); // turn off STANDBY
+  pinMode(PIN_CAN_BOOSTEN, OUTPUT);
+  digitalWrite(PIN_CAN_BOOSTEN, true); // turn on booster
+
+  Serial.begin(115200);
+  // start the CAN bus at 500 kbps
+    CAN.begin(500000);
+
+  // creating a millsDelay object using the library <millsDelay.h>
+  // This provides non-blocking delays that don't obstruct execution
+  // like using the standard delay() function would. 
+  CAN_Send_Delay.start(10); // We want to execute after every 1000 milliseconds
```

Comment:
A note on bus load. 
The bus runs at your fixed freq of 500000, that's bits per second. and remember there's the 11bit id plus the 64 bit data plus a bunch of acknowledge and arbitration bits. I don't know all the math. 

10ms -> 100Hz 
You're sending 2 msgs at 100Hz so 200 mgs/s. 

I think at 500k the can bus itself can handle 9000 8 byte msgs/s at 100% utilization but you will have timing issues at that point as messages  start to compete for priority so target around 50% utilization. Still plenty far off. 

I wouldn't suggest this yet but if you see a need for a lot more high sped messages, I would make sure your packing the messages out completely to reduce the # of messages being sent vs leaving room for expansion. 

You're nowhere near that limit. If you had 1ms timing it would start to become a major issue right now.

Reply:
That's good to know, I hadn't considered this.